packages/kbn-config-schema/src/types/number_type.test.ts

```diff
@@ -64,6 +64,20 @@ describe('#max', () => {
   });
 });
 
+describe('#unsafe', () => {
+  it('rejects unsafe numbers when false', () => {
+    expect(() =>
+      schema.number({ unsafe: false }).validate(9007199254740992)
```

Comment:
Should we repeat the test without any explicit option (implicit false)?

Reply:
Yeah, you're right 👍